Packs/IdentityRecordedFuture/Integrations/IdentityRecordedFuturePlaybookAlerts/README.md

```diff
@@ -0,0 +1,277 @@
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter**                                                     | **Description**                                                                                                                                                             | **Required** |
+    |-------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
+    | API URL (e.g., https://api.recordedfuture.com/gw/xsoar-identity/) |                                                                                                                                                                             | True         |
+    | API Token                                                         |                                                                                                                                                                             | True         |
+    | Trust any certificate (not secure)                                |                                                                                                                                                                             | False        |
+    | Use system proxy settings                                         |                                                                                                                                                                             | False        |
+    | Fetch incidents                                                   |                                                                                                                                                                             | False        |
+    | First Incidient Fetch: Time Range                                 | Limit incidents to include in the first fetch by time range. Input format: "NN hours" or "NN days". E.g., input "5 days" to fetch all incidents created in the last 5 days. | False        |
+    | Playbook Alerts: Fetched Categories                               | Some listed Playbook alert Categories might be unavailable due to limitations in the current Recorded Future subscription                                                   | False        |
+    | Maximum number of incidents per fetch                             |                                                                                                                                                                             | False        |
+    | Playbook Alerts: Fetched Statuses                                 |                                                                                                                                                                             | False        |
```

Comment:
What are the statuses? please fill the description here as well

Reply:
done